core/action/actions.go

```diff
@@ -2,7 +2,7 @@
 // Copyright 2016 Cloudbase Solutions SRL
 // Licensed under the AGPLv3, see LICENCE file for details.
 
-package actions
+package action
```

Comment:
`operation` would be a better package name. Matching the domain. We can think about it. Or split action and operation depending on need.

Reply:
Done.